### ISSUE TITLE
[layout] strange grid layout behavior

### ISSUE THREAD

**Issue by rksm** [LivelyKernel/lively.morphic#18]
I would assume that the morphs `b`, `c`, `d` would fill the space vertically:

``` js
var a = world.addMorph({name: "a", extent: pt(400,500)})
a.layout = new GridLayout({grid: [["b", "c",],
                                  ["d", "d"]]});
a.addMorph({name: "b", type: "list", fill: Color.red})
a.addMorph({name: "c", type: "list", fill: Color.green})
a.addMorph({name: "d", type: "text", fill: Color.blue})
```

But this happens:

![screen shot 2016-08-10 at 10 20 03 pm](https://cloud.githubusercontent.com/assets/467450/17569457/a84ff302-5f48-11e6-85df-8bd319c5342d.png)

**Comment by rksm** [comment]
![grid-layout-issue](https://cloud.githubusercontent.com/assets/467450/17569721/cda6bc52-5f49-11e6-89ab-7ab36b7f3677.gif)

**Comment by merryman** [comment]
fixed in: a9ffe7d198c1eecd5f9e5bb2a872c2b29e1d1b25
